footil/xcollections.py

```diff
@@ -0,0 +1,17 @@
+"""Module for collections helpers."""
+from collections.abc import Sequence
+
+
+def batch(seq: Sequence, size: int) -> Sequence:
+    """Yield sequence in batches by specified size.
+
+    Args:
+        seq (Sequence): sequence to split into batches.
+        size (int): size of a single batch.
```

Comment:
We use type hints, so we would not need to do that on docstrings. Its redundant.
```suggestion
        seq: sequence to split into batches.
        size: size of a single batch.
```